examples/usage_example.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
As an aside, how much do people look at / use the examples? I've got a more complete example (more configurable, option to use ajax spider) that we could include later..